scripts/gen.py

```diff
@@ -944,7 +944,7 @@ def parse_local_overrides(path):
 
 def generate_registrations(fgens, overrides):
   code = 'void RegisterAtenTypeFunctions() {\n'
-  code += '  auto& dispatch = at::globalATenDispatch();\n'
+  code += '  static auto dispatch = torch::RegisterOperators()\n'
```

Comment:
Why static?
This function will not be called twice anyway.
I fear that static function member initialization can play unwanted tricks here.